Dockerfile.amazon2022

```diff
@@ -1,14 +1,15 @@
-FROM amazonlinux:1 as builder
+FROM amazonlinux:2022 as builder
 
-ARG SENSU_GO_ASSET_NAME="monitoring-plugins-amazon1"
+ARG SENSU_GO_ASSET_NAME="monitoring-plugins-amazon2"
```

Comment:
you probably want the asset name to be "monitoring-plugins-amazon2022" since "monitoring-plugins-amazon2" is still valid/supported

Reply:
Fixed, forgot to update that.